src/App.vue

```diff
@@ -249,6 +251,17 @@ export default defineComponent({
     },
   },
   created() {
+    // Parse the query string into the front end.
+    const querystring = require('querystring')
```

Comment:
I don't like stray `require` sitting in the code like this. It should be at the top of the file.

Reply:
okay will change, thanks